.travis.yml

```diff
@@ -32,13 +38,16 @@ matrix:
     - name: "Aether release"
       stage: deploy
       # release only in:
-        #  branch   develop
-        #  branch   release-#.#
-        #  tag      #.#.#
-      if: (branch = develop) OR (branch =~ ^release\-[0-9]+\.[0-9]+$) OR (tag =~ ^[0-9]+(\.[0-9]+){2}$)
-      before_install:
```

Comment:
did you mean to remove this

Reply:
I moved it to the release script :grimacing: 

https://github.com/eHealthAfrica/aether/pull/738/commits/2a65f0fec83375e5230bd4b7ecd739f7a51f4731#diff-0d28f82f2d1cc0deb12146b212778bc9R81-R87

Reply:
If we finally move to GitHub actions, I prefer to have all the login in the scripts so the migration will require less steps :crossed_fingers: